gremlin-go/README.md

```diff
@@ -225,9 +225,15 @@ Please review the [staticcheck documentation][scheck docs] for more details on i
 
 ## Testing with Docker
 
-Docker allows you to test the driver without installing any dependencies. The following command can be used to run docker:
+Docker allows you to test the driver without installing any dependencies. Prior to testing in Docker, please make sure Docker is installed and running on your system, and make sure that the current Gremlin Server Docker image exist, or build it through `mvn clean install -pl :gremlin-server -DskipTests -DskipIntegrationTests=true -am && mvn install -Pdocker-images -pl :gremlin-server`.
```

Comment:
IMO this seems like a step in the wrong direction.

The tests have gotten more and more convoluted and esoteric to run, not to mention that this makes testing the *driver* dependent on a non-faulty latest build of the *server*. Why are we doing that?

If this change was only restricted to Github actions, it's still not great but at least it's automated and I'm okay with that. The fact that this changes how a user interacts with contributing to the driver unrelated to CI in a negative fashion makes it disagreeable to me.

Reply:
The dependency on the latest server is only required for the CI. 

I guess this is the wrong place to add the sentence, but it will only use the development image by default if no other server version is supplied to the `run.sh` script or through GREMLIN_SERVER in docker compose, so this is to make sure one has it.